store/use-sales-store.ts

```
@@ -0,0 +1,101 @@
+    if (agentPerformance.length === 0) return null;
+    return agentPerformance.reduce((top, agent) => 
+      agent.totalCommission > top.totalCommission ? agent : top
+    );
+  },
+  
+  getAverageConversion: () => {
+    const { agentPerformance } = get();
+    if (agentPerformance.length === 0) return '0%';
+    
+    const totalConversion = agentPerformance.reduce((sum, agent) => {
+      const rate = parseFloat(agent.conversionRate.replace('%', ''));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)